course_info/icommons.py

```diff
@@ -122,7 +121,7 @@ def _course_info_instructor_list(self, course_instance_id, **kwargs):
                 instructors = []
                 for person in rv['results']:
                     course_person_schema(person)
-                    if person.get('role')['role_id'] in INSTRUCTOR_ROLE_IDS:
+                    if person.get('role') and person.get('role')['role_id'] in INSTRUCTOR_ROLE_IDS:
```

Comment:
Elliot's right...`person.get('role', {}).get('role_id')` might *return* `None`, but it shouldn't throw an exception.